photutils/psf/tests/test_psf_photometry.py

```diff
@@ -164,6 +165,36 @@ def test_psf_photometry_gaussian():
 
 
 @pytest.mark.skipif('not HAS_SCIPY')
+def test_psf_photometry_uncertainties():
+    """
+    Make sure proper columns are added to store uncertainties on fitted
+    parameters.
+    """
+    psf = IntegratedGaussianPRF(sigma=GAUSSIAN_WIDTH)
+    f = psf_photometry(image, INTAB, psf, param_uncert=True)
+    assert_equal(f['flux_fit_uncertainty'].all() > 0 and
+                 f['x_0_fit_uncertainty'].all() > 0 and
+                 f['y_0_fit_uncertainty'].all() > 0, True)
+
+    # test for fixed params
+    psf = IntegratedGaussianPRF(sigma=GAUSSIAN_WIDTH)
+    psf.flux.fixed = True
+    f = psf_photometry(image, INTAB, psf, param_uncert=True)
+    assert_equal(f['x_0_fit_uncertainty'].all() > 0 and
+                 f['y_0_fit_uncertainty'].all() > 0, True)
+    assert_equal('flux_fit_uncertainty' in f.colnames, False)
+
+    # test in case fitter does not have 'param_cov' key
+    psf = IntegratedGaussianPRF(sigma=GAUSSIAN_WIDTH)
+    f = psf_photometry(image, INTAB, psf, fitter=SLSQPLSQFitter(),
+                       param_uncert=True)
+    assert_equal('flux_fit_uncertainty' in f.colnames or\
+                 'y_0_fit_uncertainty' in f.colnames or \
+                 'x_0_fit_uncertainty' in f.colnames, False)
```

Comment:
You should probably also test that the warning was raised. Not so much because there is a risk that it does not get raised, but because that will make sure that people who change that mechanism later think twice.
http://pytest.org/latest/recwarn.html